JetNews/app/src/main/java/com/example/jetnews/ui/JetnewsApp.kt

```diff
@@ -87,22 +89,19 @@ private fun AppDrawer(
     currentScreen: Screen,
     closeDrawer: () -> Unit
 ) {
-    FlexColumn(
-        crossAxisSize = LayoutSize.Expand
-    ) {
+    Column(modifier = ExpandedHeight wraps ExpandedWidth) {
         HeightSpacer(24.dp)
-        Padding(16.dp) {
-            Row {
-                VectorImage(
-                    id = R.drawable.ic_jetnews_logo,
-                    tint = (+MaterialTheme.colors()).primary
-                )
-                WidthSpacer(8.dp)
-                VectorImage(id = R.drawable.ic_jetnews_wordmark)
-            }
+        Row(modifier = Spacing(16.dp)) {
+            VectorImage(
+                id = R.drawable.ic_jetnews_logo,
+                tint = (+MaterialTheme.colors()).primary
+            )
+            WidthSpacer(8.dp)
+            VectorImage(id = R.drawable.ic_jetnews_wordmark)
         }
         Divider(color = Color(0x14333333))
         DrawerButton(
+            modifier = Inflexible ,
```

Comment:
it's Inflexible by default. You don't specify it in other children so should be omitted here as well